solidity/contracts/Chainlinked.sol

```diff
@@ -51,6 +51,20 @@ contract Chainlinked {
     return _run.requestId;
   }
 
+  function chainlinkRequestTo(address _oracle, ChainlinkLib.Run memory _run, uint256 _amount)
+    internal
+    returns (bytes32)
+  {
```

Comment:
Is there a way to reduce the code duplication here by calling to this function from `chainlinkRequest` and specifying the `_oracle`? Or would it not be worth it (gas)?